api/Document.json

```diff
@@ -8307,16 +8372,37 @@
             "safari_ios": {
               "version_added": false
             },
-            "webview_android": {
-              "version_added": "69",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "#enable-experimental-productivity-features",
-                  "value_to_set": "Enabled"
-                }
-              ]
-            }
+            "webview_android": [
+              {
+                "version_added": "74",
+                "notes": "<a href='https://www.chromestatus.com/feature/5190687460950016'>Chrome Platform Status</a> incorrectly describes it as <code>document.policy</code>. It was renamed in <a href='https://crbug.com/917057'>bug 917057</a> before being shipped in <a href='https://crbug.com/703703#c22'>bug 703703</a>."
+              },
+              {
+                "version_added": "73",
+                "version_removed": "74",
+                "flags": [
```

Comment:
Flags can't be turned on for WebView. Please remove.

Reply:
Thank you for prompt review!
I removed all Android WebView objects that have flags leaving only the last one. Please note that previously data had Android WebView data with flags and I removed that as well.